ansible-ipi-install/roles/installer/templates/install-config-virtualmedia.j2

```diff
@@ -72,8 +72,13 @@ platform:
     externalBridge: {{ baremetal_bridge }}
 {% endif %}
     provisioningNetwork: "Disabled"
+{% if ((release_version.split('.')[0]|int == 4) and (release_version.split('.')[1]|int == 6)) %}
     provisioningHostIP: {{ provisioningHostIP }}
```

Comment:
This change isn't mentioned in the description, does it need to be changed also?

Reply:
@derekhiggins let me now if the description is better this way and if there is anything else to modify.

If needed I can trigger another test using an inventory with the expected vars, just let me know. Thanks!